public/js/factories.js

```diff
@@ -14,14 +14,16 @@ projectMigrationApp.factory('Projects', function($http) {
 				cache : false
 			}).then(function success(result) {
 				// filter everything course sites
-				
+
 				var sourceProjects = _.filter(result.data.site_collection, 
-						function(type){ 
-					return type != 'course' && type != 'specialized_courses'; 
+					function(site){ 
+					return (site.type === 'project' || site.type==='myworkspace');
```

Comment:
It should be any type other than "course" related. It is not limited to 'project', or 'myworkspace'